### ISSUE TITLE
[Question] Restrict bundling to specific Node.js versions

### ISSUE THREAD

**Issue by ecrmnn** [JeffreyWay/laravel-mix#2163]
We're an agency with 10 developers and we run *a lot* of Laravel sites. We use Laravel Mix on basically all projects. Switching projects/developers sometimes leads to bundling errors. E.g. someone runs `npm install` with the wrong Node.js or npm version. Using `nvm` and a `.nvmrc` file in the project root usually solves this, but its nothing stopping you from ignoring the file.

To solve this, we made a small package last week, [`nvm-guard`](https://github.com/ecrmnn/nvm-guard). 
Nvm Guard exits the Node process if the version in `.nvmrc` is not met.
We use it like this in package.json:
```json
{
  "scripts": {	
    "hot": "nvm-guard && cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
  }
}
```

Sorry for the long text, but I thought this needed a real world example.

**TL;DR**
Do you think Laravel Mix should have an option for adhering to the Node.js version specified in `.nvmrc` or simply specify a version?

Is this within the scope of the project or should this be solved by a plugin?

Example
```js
const mix = require('laravel-mix');

mix.useNvmrc();
// or
mix.useNodeVersion('8.10.0')
```
